client/src/components/ArtistProfile.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styledComponents from './styledArtist';
+import PremiumPlan from './premiumPlan.jsx';
+
+class ArtistProfile extends React.Component {
+  constructor(props) {
+    super(props);
+    const { artist } = props;
+    this.state = {
+      artist,
+    };
+  }
+
+  componentWillReceiveProps(nextProps) {
+    const { artist } = this.props;
+    console.log(nextProps.artist);
+    if (nextProps.artist !== artist) {
+      this.setState({ artist: nextProps.artist });
+    }
+  }
+
+  render() {
+    const { artist } = this.state;
+    const { Avatar } = styledComponents;
```

Comment:
I think you can assign all of these at once by doing { Avatar, ProfileDiv, ArtistContentDiv, etc } = styledComponents